src/configs/latest.ts

```diff
@@ -47,6 +47,20 @@ export const rules = {
     }],
     "typeof-compare": true,
     "unified-signatures": true,
+
+    // added in v4.4
+    "arrow-return-shorthand": true,
+    "no-unbound-method": true,
```

Comment:
We haven't enabled any other rules that require type information in this config and I don't think we should start now. We could introduce a separate config for type checking rules and perhaps reconsider for the next major release.